modules/nextflow/src/main/groovy/nextflow/Const.groovy

```diff
@@ -55,12 +57,12 @@ class Const {
     /**
      * The app build time as linux/unix timestamp
      */
-    static public final long APP_TIMESTAMP = 1552584379498
+    static public final long APP_TIMESTAMP = 1553784415284
 
```

Comment:
I ideally `APP_TIMESTAMP` and `APP_BUILDNUM` should never be committed (in this case you should commit the `ISO_8601_DATETIME_FORMAT`) otherwise it conflicts when merging it.

Reply:
ah, this got overwritten during compilation for testing purposes and committed, cause I introduced a new constant.